projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -1143,11 +1124,139 @@ function grunion_feedback_admin_notice() {
+		add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_scripts' ) );
+		add_action( 'admin_footer-edit.php', array( $this, 'print_export_modal' ) );
+
+		if ( defined( 'JETPACK_BETA_BLOCKS' ) && JETPACK_BETA_BLOCKS || ( defined( 'IS_WPCOM' ) && IS_WPCOM && is_automattician() ) ) {
+			add_action( 'wp_ajax_grunion_export_to_gdrive', array( $this, 'export_to_gdrive' ) );
+		}
+	}
+
+	/**
+	 * Hook handler for admin_enqueue_scripts hook
+	 */
+	public function admin_enqueue_scripts() {
+		$current_screen = get_current_screen();
+		if ( ! in_array( $current_screen->id, array( 'edit-feedback', 'feedback_page_feedback-export' ), true ) ) {
+			return;
+		}
+		add_thickbox();
+		wp_localize_script( 'grunion-admin', 'exportParameters', array( 'exportError' => esc_js( __( 'There was an error exporting your results', 'jetpack' ) ) ) );
+	}
+
+	/**
+	 * Prints the modal markup with export buttons/content.
+	 */
+	public function print_export_modal() {
+		if ( ! current_user_can( 'export' ) ) {
+			return;
+		}
+
+		// if there aren't any feedbacks, bail out
+		if ( ! (int) wp_count_posts( 'feedback' )->publish ) {
+			return;
+		}
+
+		$current_screen = get_current_screen();
+		if ( ! in_array( $current_screen->id, array( 'edit-feedback', 'feedback_page_feedback-export' ), true ) ) {
+			return;
+		}
+
+		$jetpack_logo = new Logo();
+		?>
+		<div id="feedback-export-modal" style="display: none;">
+			<div class="feedback-export-modal__wrapper">
+				<div class="feedback-export-modal__header">
+					<h1 class="feedback-export-modal__header-title"><?php echo esc_html( __( 'Export your Form Responses', 'jetpack' ) ); ?></h1>
+					<p class="feedback-export-modal__header-subtitle"><?php echo esc_html( __( 'Choose your favorite file format or export destination:', 'jetpack' ) ); ?></p>
+				</div>
+				<div class="feedback-export-modal__content">
+					<?php $this->get_csv_export_section(); ?>
+					<?php
+					if ( defined( 'JETPACK_BETA_BLOCKS' ) && JETPACK_BETA_BLOCKS || ( defined( 'IS_WPCOM' ) && IS_WPCOM && is_automattician() ) ) {
+						$this->get_gdrive_export_section();
+					}
+					?>
+				</div>
+				<div class="feedback-export-modal__footer">
+					<div class="feedback-export-modal__footer-column">
+						<a href="https://jetpack.com/support/jetpack-blocks/contact-form/" title="Jetpack Forms" rel="noopener noreferer" target="_blank" class="feedback-export-modal__footer-link">
+							<?php echo $jetpack_logo->get_jp_emblem(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
+						</a>
+						<a href="https://jetpack.com/support/jetpack-blocks/contact-form/" title="Jetpack Forms" rel="noopener noreferer" target="_blank" class="feedback-export-modal__footer-link">Jetpack Forms</a>
+					</div>
+					<div class="feedback-export-modal__footer-column">
+						<a href="https://automattic.com" title="Automattic" rel="noopener noreferer" target="_blank" class="feedback-export-modal__footer-link">
```

Comment:
You can probably make those strings translatable as well.

Reply:
You mean "Jetpack Forms"? I thought it was some kind of "trademark", not to be translated. I'll add it and trust the translators on it.

Reply:
I would translate it, since "Forms" may not mean much in some languages. We can, however, add context to the strings so translators know how to translate (i.e. not translate it if their language can work that way).
I've pushed 3e642eccae6139cd99b066167d8c3a52f20b5509 to do just that.